test/ComponentRouter-test.js

```diff
@@ -1,23 +1,160 @@
+import React from 'react/addons';
+import createComponent from './utils/createComponent';
+import FooBar from '../src/example/FooBar/FooBar';
```

Comment:
Yeah good idea. 

Interestingly (or not) I was trying to use the actual `<Chart />` Component from Quickstart with the inject-loader, much like it was used with `ComponentRouter` and others, but it would still throw errors for the CSS file - unlike what happened if I tried the same thing in the main codebase. I saw you had a bit of input in the `inject-loader` repo for similar stuff with ES6 imports. Not sure if its an `inject-loader` bug or not? (we dont use css-loader/style-loader in the webpack test config)...sorry for the digression.